client/js/pages/archive/index.js

```diff
@@ -38,8 +38,8 @@ const archivePage = (state, prev, send) => {
             <div id="permissions" class="dat-detail">${permissions({owner: state.archive.instance ? state.archive.instance.owner : false})}</div>
             <div id="hyperdrive-size" class="dat-detail"><p id="size">${prettyBytes(state.archive.size || 0)}</p></div>
             <div id="peers" class="dat-detail">${state.archive.numPeers} Source(s)</div>
-            <div style="display: none;" id="speed" class="dat-detail dat-detail--speed"><div>
-              <span id="download-speed"></span> / <span id="upload-speed"></span>
+            <div style="display: block;" id="speed" class="dat-detail dat-detail--speed"><div>
```

Comment:
i think we decided no inline styles. also, is display:block even necessary on this div?